app/legacy_lib/maintain_payments_where_supporter_is_gone.rb

```diff
@@ -103,7 +103,7 @@ def self.cleanup(sorted_by_kind, api_key)
           
         end
       end
-      return {manual_payments: manual_payments, manual_refunds: manual_refunds, manual_disputes: manual_disputes}
+      {manual_payments: manual_payments, manual_refunds: manual_refunds, manual_disputes: manual_disputes}
```

Comment:
This is an maintenance method without a spec. I'm not worried that we don't have a spec to test this change.